website/static/js/filepage/index.js

```diff
@@ -147,8 +147,8 @@ var FileViewPage = {
         //Ran into problems with mithrils caching messing up with multiple "Panels"
         self.revisions = m.component(FileRevisionsTable, self.file, self.node, self.enableEditing, self.canEdit);
         self.revisions.selected = false;
-        self.revisions.title = 'Revisions';
-
+        self.revisions.title = 'More';
```

Comment:
For a thought, this one line should be changed in its own PR. Calling the panel that currently just hosts revisions restricts the addition of more functionality that we don't want to clog the top bar with.